pkgs/applications/audio/aeolus/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libclthreads, zita-alsa-pcmi, alsaLib, libjack2,
+  libclxclient, libX11, libXft, readline}:
```

Comment:
nitpick: please put the "," on the start of the line and a space before the "}".